actioncable/lib/action_cable/subscription_adapter/postgresql.rb

```diff
@@ -0,0 +1,103 @@
+require 'thread'
+
+begin
+  require 'pg'
+rescue Gem::LoadError => e
+  raise Gem::LoadError, "You are trying to use the PostgreSQL ActionCable adapter, but do not have the proper gems installed. Add `gem 'pg'` to your Gemfile (and ensure its version is at the minimum required by ActionCable)."
```

Comment:
When we aren't referring to the library's constant, there's a space in the name: Action Cable, Active Record etc.